File: solutions/csharp/etl/1/Etl.cs
using System;
using System.Collections.Generic;

public static class Etl
{
    public static Dictionary<string, int> Transform(Dictionary<int, string[]> old)
    {
        var ret = new Dictionary<string, int>();
        foreach(var kvp in old) {
            foreach(var v in kvp.Value) {
                ret[v.ToLower()] = kvp.Key;
            }
        }
        return ret;
    }
}

File: solutions/swift/etl/1/Sources/Etl/Etl.swift
//Solution goes in Sources

class ETL {
    static func transform(_ old:[Int:[String]]) -> [String:Int] {
        var ret = [String:Int]()
        for kvp in old {
            for v in kvp.value {
                ret[v.lowercased()] = kvp.key
            }
        }
        return ret
    }
}
